### DISCUSSION TITLE
JR 8911HV Servos Problems after update 1.5.7 to 1.5.13

### DISCUSSION THREAD

**Discussion by TheLeoDaluz** [FrSkyRC/ETHOS-Feedback-Community#4425] [General]
Hi guys.

I´m having some problems with my Edge SkyWing 42% JR 8911HV Elevator Servos.

After update my radio from the ethos 1.5.7 to 1.5.13 my servos just don´t stop moving all the way by themselves.
 
I´ve never had this problems with this type of servos but now i´m.
 
I´ve already try to downgrade my ethos using the Ethos Suit Program to the 1.5.7 but everthing goes wrong and now i got back to the 1.5.13.

PS: I´ve already update the receiver firmware too but nothing new on it.

Coulf you guys please help me with?.

By now, i just cant fly anymore my plane because its just impossible to control him now as the servos from my elevator just DON´T stop to move by themselves.

Huge thanks and really hope you guys can help me with.

**Comment by Dsrenger** [comment]
First, try to power off telemetry.

If they stop moving, then the telemetry powering into the signal wire of the servo is the real reason. 

If that helps, switch the telemetry power to 25mW and see if it is gone.

**Comment by TheLeoDaluz** [comment]
@Dsrenger i´ve already tried both options that you suggest but none of them give me something different.
